rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java

```diff
@@ -139,8 +139,9 @@ private CachingRlsLbClient(Builder builder) {
             timeProvider);
     RlsRequestFactory requestFactory = new RlsRequestFactory(lbPolicyConfig.getRouteLookupConfig());
     rlsPicker = new RlsPicker(requestFactory);
-    ManagedChannelBuilder<?> rlsChannelBuilder =
-        helper.createResolvingOobChannelBuilder(rlsConfig.getLookupService());
+    ManagedChannelBuilder<?> rlsChannelBuilder = helper.createResolvingOobChannelBuilder(
+        rlsConfig.getLookupService(), helper.getUnsafeChannelCredentials());
```

Comment:
Add a comment describing why it is safe to use the unsafe credentias.

Reply:
Done.